charts/caluma/values.yaml

```diff
@@ -41,6 +41,10 @@ frontend:
   ## ref: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
   # - name: EXAMPLE
   #   value: example
+  service:
+    name: "caluma-portal-frontend"
```

Comment:
we should generate the name from the helm release name and not create it with this values. 

This will make sure we can have two separate caluma releases in the same namespace.

Reply:
Let's ask about this change @derrabauke , can be done post PR